src/TransIpDynDns/Settings.php

```diff
@@ -9,59 +9,59 @@
 final class Settings
 {
     const DEFAULT_TTL = 300;
+    const DEFAULT_IPV6 = true;
 
     /**
-     * @var string
+     * @var array
      */
-    private $domain;
+    private $records;
 
     /**
-     * @var string
+     * @var int
      */
-    private $subdomain;
+    private $ttl;
 
     /**
-     * @var int
+     * @var bool
      */
-    private $ttl;
+    private $ipv6;
```

Comment:
I think the name `ipv6` here is too ambiguous, as it could imply quite a lot of different kind of values. Since it its a boolean indicating whether or not IPv6/AAAA-records are pushed, may i suggest renaming this property to something along the lines of `ipv6Enabled` or `useIpv6`?

The corresponding function `getIpv6` can than be renamed to something like `isIpv6Enabled` or `shouldUseIpv6`.